posthog/api/property_definition.py

```diff
@@ -33,10 +34,26 @@ class PropertyDefinitionViewSet(
     permission_classes = [permissions.IsAuthenticated, OrganizationMemberPermissions]
     lookup_field = "id"
     ordering = "name"
-    filter_backends = [filters.SearchFilter]
-    search_fields = ["name"]
 
     def get_queryset(self):
+        # input query
+        search_string = self.request.GET.get("search", None)
```

Comment:
**Blocking**. I don't think these changes make sense because #5039 is coming in.